unison-cli/unison/ArgParse.hs

```diff
@@ -457,18 +468,26 @@ fileArgument varName =
 transcriptParser :: Parser Command
 transcriptParser = do
   -- ApplicativeDo
+  shouldSaveCodebaseTo <- saveCodebaseToFlag
   shouldSaveCodebase <- saveCodebaseFlag
   mrtsStatsFp <- rtsStatsOption
   files <- liftA2 (NE.:|) (fileArgument "FILE") (many (fileArgument "FILES..."))
-  pure (Transcript DontFork shouldSaveCodebase mrtsStatsFp files)
+  pure (let saveCodebase = case shouldSaveCodebaseTo of
+                              DontSaveCodebase -> shouldSaveCodebase
+                              _ -> shouldSaveCodebaseTo
+        in Transcript DontFork saveCodebase mrtsStatsFp files)
```

Comment:
I'm sure there's a more idiomatic way of doing this, but I kept running into `no instance of (Monad Parser)` found whenever I tried 😭

Reply:
Yeah, ApplicativeDo notation can be a bit finicky; this seems fine though 👍🏼